tests/Unit/Domain/Python/Test_JacobianDiagnostic.py

```diff
@@ -6,14 +6,10 @@
 
 
 def jacobian_diagnostic(analytic_jacobian, numeric_jacobian_transpose):
-    analytic_sum = analytic_jacobian.sum(axis=0)
-    numeric_sum = numeric_jacobian_transpose.sum(axis=1)
-    for i, x in enumerate(analytic_sum):
-        if abs(x) < 1.e-10 and abs(numeric_sum[i]) < 1.e-10:
-            analytic_sum[i] = 0.0
-            numeric_sum[i] = 0.0
-    jac_diff_sum = 1.0 - analytic_sum / (numeric_sum + 1.e-20)
-    jac_diag = np.sqrt(np.dot(jac_diff_sum, jac_diff_sum))
+    analytic_sum = (analytic_jacobian * analytic_jacobian).sum(axis=0)
```

Comment:
`analytic_jacobian**2`

Reply:
I'm no longer using RMS, but I'll keep in mind that **2 is ok to use in future python tests